draft-ietf-rats-architecture.md

```diff
@@ -1159,7 +1157,10 @@ an Attestation Result for any other purpose.
 ## Attester and Attestation Key Protection
 
 
-Implementers need to pay close attention to the isolation and protection of the Attester and the factory processes for provisioning the Attestation key material. If either of these are compromised, the remote attestation becomes worthless because the attacker can forge Evidence.
+Implementers need to pay close attention to the protection of the Attester and the factory processes for provisioning the Attestation key material. If either of these are compromised, the remote attestation becomes worthless because the attacker can forge Evidence.
```

Comment:
```suggestion
Implementers need to pay close attention to the protection of the Attester and the factory processes for provisioning the Attestation key material. If either of these are compromised, the remote attestation becomes worthless because an attacker can forge Evidence.
```